src/codeu/chat/server/Controller.java

```diff
@@ -92,6 +92,13 @@ public ConversationHeader newConversation(String title, Uuid owner) {
     return newConversation(id, title, owner, time);
   }
 
+  public ConversationHeader newPlayConversation(Uuid member, String title) {
```

Comment:
Since this function is not used anywhere else apart from this class, it's a helper function. So you can make it private static.